src/annotator/config/index.js

```diff
@@ -22,6 +22,13 @@ export default function configFrom(window_) {
     enableExperimentalNewNoteButton: settings.hostPageSetting(
       'enableExperimentalNewNoteButton'
     ),
+    enableExperimentalPDFSideBySide: settings.hostPageSetting(
```

Comment:
The naming of this config is up for all sorts of discussion. Given that we want to be more abstract about document types, there could be, e.g., an array of document types for which we should lay out side-by-side. I didn't want to overthink it for now. A single config option makes this easy to enable and test in a variety of environments.